examples/ipam/main.tf

```diff
@@ -1,6 +1,5 @@
 module "vpc" {
-  source  = "aws-ia/vpc/aws"
```

Comment:
for each example, can you flip it back from `source = "../.." to the module source in tf regstiry